examples/hooks/App.css

```diff
@@ -16,6 +16,36 @@ body {
   margin: 0 auto;
 }
 
+.Container {
+  display: grid;
+  align-items: flex-start;
+  grid-template-columns: 200px 1fr;
+  gap: 0.5rem;
+}
+
+.Search {
+  display: grid;
+  gap: .5rem;
+}
+
+.Search-header {
+  display: flex;
+}
+
+.Search-box {
+  flex: 1;
+  margin-right: .5rem;
```

Comment:
Since we use `gap` in `.Search`, we can also use it in `.Search-header`?

Reply:
Indeed! I adressed it in 6c516d5c. The grid layout removed the need to add styles for SearchBox, so that's also handled as well.